src/scripts/run-dast.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+cd /zap/ || return
+if [  "$SOOS_PROJECT_NAME" == "" ]; then
```

Comment:
Consider replacing this with:

```
if [ -z "$SOOS_PROJECT_NAME" ]; then
```

It's a more reliable way of checking empty strings in bash.

Reply:
I've ended up using `-n` since i think that's what I needed in this case.